peerberrypy/api.py

```diff
@@ -575,10 +575,10 @@ def get_transactions(
     def get_mass_transactions(
             self,
             quantity: int,
-            start_date: date,
-            end_date: date,
-            transaction_types: list = None,
-            periodicity: str = None,
+            start_date: Optional[date],
+            end_date: Optional[date],
+            transaction_types: Optional[list[str]] = None,
+            periodicity: Optional[str] = None,
             sort: str = 'amount',
```

Comment:
Instead of having `list[str]`, it's better to have `List[str]` to ensure backward compatibility with older Python versions. You can use the List from the typing module.